frontend/packages/metal3-plugin/src/selectors/baremetal-hosts.ts

```diff
@@ -18,6 +18,8 @@ import {
 
 export const getHostOperationalStatus = (host: BareMetalHostKind): string =>
   _.get(host, 'status.operationalStatus');
+export const getHostErrorType = (host: BareMetalHostKind): string =>
+  _.get(host, 'status.errorType', '');
```

Comment:
Returning undefined is fine.
```suggestion
  _.get(host, 'status.errorType');
```